plugins/templates/lmgtfy.jinja2.md

```diff
@@ -1 +1 @@
-Hey! You seem to have asked a question which could have been solved by other means as well. Click [this](https://www.lmgtfy.com/?q={{ query }}). When you ask a question, be sure to try out a few things first - you're in a much better position to help yourself than we are. Try googling, thinking, searching on GitHub or [git grep](https://git-scm.com/docs/git-grep) if you are looking for source code. [StackOverflow](http://stackoverflow.com/help/how-to-ask) has a good guide on how to ask questions in a way that allows us to help you even faster!
+Hey! You seem to have asked a question which could have been solved by other means as well. Click [this](https://www.lmgtfy.com/?q={{ query }}). When you ask a question, be sure to try out a few things first - you're in a much better position to help yourself than we are. Try googling, thinking, searching on GitHub or [git grep](https://git-scm.com/docs/git-grep) if you are looking for source code.Coala now has a fancy [coala](https://stackoverflow.com/questions/tagged/coala) StackOverflow tag, you can use it in [StackOverflow](http://stackoverflow.com/help/how-to-ask) to ask questions in a way that allows us to help you even faster!
```

Comment:
Add a space after `... if you are looking for source code.` Also, coala is always written with a small 'c', even if it is the first word of a sentence.